plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/editparts/AbstractFBNElementEditPart.java

```diff
@@ -388,7 +390,9 @@ private static int getInterfaceInputElementIndex(final InterfaceEditPart interfa
 		}
 		if (interfaceEditPart.isVariable()) {
 			if (((VarDeclaration) interfaceEditPart.getModel()).isInOutVar()) {
-				return interfaceList.getInOutVars().indexOf(interfaceEditPart.getModel());
+				final List<VarDeclaration> visInOutList = interfaceList.getInOutVars().stream()
```

Comment:
It would be better to make a method for that. 
I think you are using that multiple times.

please don't name a variable x. 
it should have a meaningful name